src/components/HeroDecorative/HeroDecorative.scss

```diff
@@ -0,0 +1,97 @@
+@import '../shared/main.scss';
```

Comment:
You should be adding specific things here like responsive utilities or fonts instead of adding the main.scss file.